cache_images/fedora_packaging.sh

```diff
@@ -174,6 +174,10 @@ if [[ "$PACKER_BUILD_NAME" =~ fedora ]]; then
         python3-requests
         python3-requests-mock
     )
+else # podman-sequoia is only available in Rawhide
+    INSTALL_PACKAGES+=( \
+        podman-sequoia
+    )
```

Comment:
here it would be good to add a timebomb as reminder to add sequoia to the older versions as well once available. I guess if it is only rawhide for now it should be set to November once f43 is released at least.